builder/vmware/common/driver_esx5.go

```diff
@@ -45,11 +53,66 @@ type ESX5Driver struct {
 	VMName         string
 	CommConfig     communicator.Config
 
+	ctx    context.Context
+	client *govmomi.Client
+	finder *find.Finder
+
 	comm      packer.Communicator
 	outputDir string
 	vmId      string
 }
 
+func NewESX5Driver(dconfig *DriverConfig, config *SSHConfig, vmName string) (Driver, error) {
```

Comment:
There was a user in one of the issues who commented that the vsphere url was readonly if you were on a free non-licensed standalone esx instancce. Are we sure that needing to access the client via this url won't break the builder for those users?

Reply:
Just tested it here and you're right, it doesn't work :|

Reply:
I think we won't be able to support non-licensed anymore for ESXI6.5+ instances. The VNC-over-websocket idea is also possible to implement via API and all of the other options mentioned in the vmware doc here https://docs.vmware.com/en/VMware-vSphere/7.0/rn/vsphere-esxi-vcenter-server-70-release-notes.html requires a paid license. 

What I'm going to do is to make all of this code work behind a flag, this way I'll keep VNC as default for esxi6.0 and lower and whoever needs the USB scancodes and has a paid license will be able to enabled it.

Reply:
Actually, I was just able to acquire a websocket ticket to the VNC connection for a free licensed ESXi host 🤔  I will work more on that!

Reply:
The challenge that I had when looking into the VNC connection was that it would die during the VNC handshake process after getting the ticket. Here was what I was doing: https://github.com/hashicorp/packer/issues/8984#issuecomment-611183760

Reply:
I'm trying that as well but the protocols are giving me wrong versions, so I'm struggling to find the right ones. I don't know why but `vmware-vvc` and `binary` can't be parsed for me.  Any thoughts why?

Reply:
It would read the protocol versions, send in the version, but not read the security handshake.

Reply:
I'm not sure, the version response that I was getting on 6.7 was correct.